package.json

```diff
@@ -1436,12 +1436,77 @@
                 "packageArtifactId": "azure-cognitiveservices-qnamaker"
             },
             {
+                "packageArtifactId": "azure-spring-cloud-starter-storage-queue",
+                "packageVersion": "2.0.0-beta.1",
+                "packageGroupId": "com.azure.spring",
+                "inputPath": [],
                 "packageDownloadUrl": "https://repo1.maven.org/maven2",
+                "excludePath": []
+            },
+            {
+                "packageArtifactId": "azure-spring-cloud-storage",
+                "packageVersion": "2.0.0-beta.1",
                 "excludePath": [],
-                "packageVersion": "1.0.0-beta.2",
+                "packageGroupId": "com.azure.spring",
+                "packageDownloadUrl": "https://repo1.maven.org/maven2",
+                "inputPath": []
+            },
+            {
+                "packageDownloadUrl": "https://repo1.maven.org/maven2",
                 "inputPath": [],
-                "packageGroupId": "com.azure",
-                "packageArtifactId": "azure-security-keyvault-jca"
```

Comment:
Why do we remove the package? 
Sorry, I did not see we update the version to 1.0.0-beta.2 already. I think 1.0.0-beta.2 is working version. If we want to remove we have to remove the config in unified-reference as well

Reply:
Oh yeah, you are right, the 1.0.0-beta.2 has fixed the dependency problem we discussed before, I read it wrong, too.